Document all breaking changes. (#846)
telerik/maui-docs: knowledge-base/trial-message.md

---

> title: See Trial Message When Having a Commercial License

Reviewer: ```suggestion
title: Removing the Trial Message after Purchasing a License
```

> page_title: Trial Message When Having a Commercial License - .NET MAUI Knowledge Base

Reviewer: ```suggestion
page_title: Removing the Trial Message after Purchasing a License
```

> description: I See the Trial Message even I have installed the commercial license.

Reviewer: ```suggestion
description: I still see the trial message even though I have purchased and installed the commercial license.
```

type: troubleshooting
slug: trial-message-commercial-license
res_type: kb
---

## Environment

| Version | Product | Author | 
| --- | --- | ---- | 
| All versions | Telerik UI for .NET MAUI |[Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 

## Description

> While using a trial license, a trial message is rendered over the components and there is an alter with the following message:

Reviewer: ```suggestion
While using a trial license, a trial message is rendered over the components and you see a popup window:
```

![Telerik .NET MAUI Trial Message](images/trial-message.png)

> If you have a commercial license, but you still see the trial message, try the following steps:

Reviewer: ```suggestion
If you have a commercial license, but you still see the trial message:
```

1. Ensure that the licensed package is referenced in the project (`Telerik.UI.for.Maui` instead of `Telerik.UI.for.Maui.Trial`).

> 1. Uninstall any Trial installations from the machine.

Reviewer: ```suggestion
1. Uninstall any installed trial versions from the machine.
```
